README.md

```diff
@@ -81,30 +81,43 @@ test('Emulate user actions and perform a verification', async t => {
 Additionally, TestCafe automatically generates source maps for easy debugging.
 To debug your test code, start a debugging session in an IDE that supports source maps.
 
-### Direct Access to Page Elements
+### Flexible Selectors System
 
-TestCafe allows you to access webpage elements using standard CSS selectors or [custom selectors](https://devexpress.github.io/testcafe/documentation/test-api/selecting-page-elements/selectors.html) that run client JavaScript code.
-You can call a custom selector as a regular function within your test.
-It will execute your code on the client and pass the returned value back to the test.
-This allows you to determine the state of each element on the tested page or select a proper element to perform an action on.
+TestCafe supports a flexible selector system that provides API with rich capabilities for writing test scripts.
+You can access webpage elements in different ways using [selectors](https://devexpress.github.io/testcafe/documentation/test-api/selecting-page-elements/selectors.html). For example, you can use one of the following selector types:
+
+* CSS selector
+* Text
+* DOM hierarchy
+* or use any custom logic
+
+The selector API provides methods that can be combined together, thus providing you with a flexible functional-style selector mechanism.
+
+```js
+const macOSInput = Selector('.column').find('label').withText('MacOS').child('input');
+```
+
+You can also use the helpful and popular Page Object methodology. This methodology is based on wrapping the pages or page fragments of a web app into objects:
```

Comment:
Here and on landing page: make `Page Object` a link to http://martinfowler.com/bliki/PageObject.html. Remove the following sentence `This methodology is based on wrapping the pages...`, since link describes PageObject in details